src/main/java/vazkii/botania/common/item/relic/ItemLokiRing.java

```diff
@@ -61,13 +66,32 @@ public class ItemLokiRing extends ItemRelicBauble implements IExtendedWireframeC
 	private static final String TAG_Y_ORIGIN = "yOrigin";
 	private static final String TAG_Z_ORIGIN = "zOrigin";
 	private static final String TAG_MODE = "mode";
+	private static final String TAG_BREAKING_MODE = "breaking";
 	private boolean recursion = false;
 
 	public ItemLokiRing() {
 		super(LibItemNames.LOKI_RING);
 		MinecraftForge.EVENT_BUS.register(this);
 	}
 
+	@SubscribeEvent
+    public void onBlockBreak(BlockEvent.BreakEvent event) {
+        EntityPlayer player = event.getPlayer();
+
```

Comment:
Several indentation issues in this method, please correct them.

Reply:
Almost good, except the method is now unaligned with the rest of the file (and the annotation).

Reply:
It looked aligned in my editor, fixed